packages/workflows-shared/package.json

```diff
@@ -0,0 +1,60 @@
+{
+	"name": "@cloudflare/workflows-shared",
+	"version": "0.5.4",
+	"description": "Package that is used at Cloudflare to power some internal features of Cloudflare Workers.",
```

Comment:
is this description vague on purpose? just asking

Reply:
No that last word is meant to say Workflows 🤦